inspector/src/components/graph/canvasGraphService.ts

```diff
@@ -138,13 +178,13 @@ export class CanvasGraphService {
             bounds.end = this._getClosestPointToTimestamp(dataset, bufferMaximum) + 1;
 
             // keep track of largest timestamp value in view!
-            globalTimeMinMax.max = Math.max(dataset.data[bounds.end - 1].timestamp, globalTimeMinMax.max);
+            this._globalTimeMinMax.max = Math.max(dataset.data[bounds.end - 1].timestamp, this._globalTimeMinMax.max);
         });
 
-        let updatedScaleFactor = Scalar.Clamp((globalTimeMinMax.max - globalTimeMinMax.min)/(bufferMaximum - globalTimeMinMax.min), 0.8, 1); 
+        let updatedScaleFactor = Scalar.Clamp((this._globalTimeMinMax.max - this._globalTimeMinMax.min)/(bufferMaximum - this._globalTimeMinMax.min), 0.8, 1); 

```

Comment:
Could we move the 0.8 as a constant ?

Reply:
Yep! its basically just bound between the original scaleFactor of 0.8 and 1 so i will update that!

Reply:
Done!